mapie/control_risk/crc_rcps.py

```diff
@@ -0,0 +1,204 @@
+import numpy as np
+from numpy.typing import NDArray
+from typing import Optional, Tuple
+
+sigma_init = .25  # Value given in the paper [1].
+
+
+def _get_r_hat_plus(
+    risks: NDArray,
+    lambdas: NDArray,
+    method: Optional[str],
+    bound: Optional[str],
+    delta: Optional[float],
```

Comment:
```suggestion
    method: str,
    bound: str,
    delta: float,
```

Reply:
can't do that as,  in MultilabelClassifier theses attributes have type Optional[float].

Reply:
The docstring must therefore be adapted.

Reply:
@SZiane Please, don't forget to precise that the 3 parameters method, bound and delta are Optional in the docsting.